mq/plugins/broFixup.py

```diff
@@ -2,8 +2,14 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 # Copyright (c) 2017 Mozilla Corporation
+#
+# Contributors:
```

Comment:
We no longer keep contributor lists in each file, so you'll want to remove lines 5 - 9

Reply:
you might want to just add a contributor file at the root (common practice since its more discoverable than commits and make people happy to be recognized)

Reply:
I'd disagree, I think the list of contributors by commit is sufficient here, as it stays updated automatically.